### DISCUSSION TITLE
Possible to use a mix of iOS default menuItems and custom menuItems?

### DISCUSSION THREAD

**Discussion by alyxb** [react-native-webview/react-native-webview#3280] [Q&A]
I'm currently using the `menuItems` param to pass custom menu items:
https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#menuitems

Ideally, I'll like to keep the default `lookup` iOS menu item, but when passing the custom `menuItems` the default iOS options are not available anymore. 

I've also tried using the `suppressMenuItems` param, excluding all but the `lookup` item, and including my custom menuItems, but the custom menuItems still overrides the iOS menu. 

Example, where I suppress all but `lookup` and add custom menu items:

```
    suppressMenuItems={[
      'cut',
      'copy',
      'paste',
      'delete',
      'select',
      'selectAll',
      'replace',
      'translate',
      'bold',
      'italic',
      'underline',
      'share',
    ]}
    menuItems={[
        { label: 'Highlight', key: 'highlight' },
        { label: 'Copy', key: 'copy' },
    ]}

```

Is this inability to blend menu item options an issue at the `UIMenuController` level, or could be implemented in the `react-native-webview` library?

Or is there a way to pass a `lookup` method from `onCustomMenuSelection` to the default iOS call?

**Comment by admbtlr** [comment]
I'm trying to do this too. Did you figure it out @alyxb ?

**Reply by alyxb** [reply]
@admbtlr I did not, I believe it would require lower level code modification. My workaround was to add my own custom version of each menu feature.
